inference-engine/src/transformations/include/transformations/op_conversions/convert_gather_0d.hpp

```diff
@@ -22,7 +22,7 @@ class TRANSFORMATIONS_API ConvertGather0D;
 
 /**
  * @ingroup ie_transformation_common_api
- * @brief ConvertGather0D decomposes v1::Gather operation into v0::Unsqueeze + v1::Gather + v0::Squeeze pattern when gather indices is scalar
+ * @brief ConvertGather0D decomposes v7::Gather operation into v0::Unsqueeze + v7::Gather + v0::Squeeze pattern when gather indices is scalar
```

Comment:
This transformation is not used in CommonOptimizations, so I suggest to revert changes for this transformation.

Reply:
fixed